src/line-chart/interfaces.ts

```diff
@@ -24,6 +24,13 @@ export interface LineChartProps<T extends ChartDataTypes>
    * See the usage guidelines for more details.
    */
   emphasizeBaselineAxis?: boolean;
+
+  /**
+   * Specifies custom rendering of the series displayed in the chart popover.
+   * Use this for wrapping keys or values in links, or to display an additional
+   * level of nested items.
+   */
+  detailPopoverSeriesContent?: MixedLineBarChartProps.DetailPopoverSeriesContent<T>;
```

Comment:
Can we reuse this via CartesianChartProps? As we don't provide the feature yet for the AreaChart we can exclude it from there with:

```
// Omitting detailPopoverSeriesContent because the feature is not yet supported ...
interface AreaChartProps<T> extends Omit<CartesianChartProps<T>, 'detailPopoverSeriesContent'>{ ... }
```